include/secp256k1_musig.h

```diff
@@ -190,7 +184,7 @@ SECP256K1_API int secp256k1_musig_partial_sig_parse(
  *
  *  Different orders of `pubkeys` result in different `agg_pk`s.
  *
- *  The pubkeys can be sorted before combining with `secp256k1_xonly_sort` which
+ *  The pubkeys can be sorted before combining with `secp256k1_pubkey_sort` which
```

Comment:
nit:
```suggestion
 *  Before combining, the pubkeys can be sorted with `secp256k1_pubkey_sort`, which
```
My brain parsed this as "(The pubkeys can be sorted (before combining with `secp256k1_xonly_sort`)) which ..."

Reply:
Done (and replaced "combine" with "aggregate")